manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -6,9 +6,10 @@ package app
 import (
 	"context"
 	"fmt"
-	"os"
+
 	"strings"
 	"time"
+	"os"
```

Comment:
formatting issues?

Reply:
I don't understand what you mean?

Reply:
Usually, imports are sorted in the lexicographical order, but not important.